### PR TITLE
Jdaviz doc

### PR DESCRIPTION
**Pull request by PatrickOgle**
Documentation for Cubeviz plugins

### CODE REVIEW

File: docs/cubeviz/plugins.rst [hunk 1]
```diff
@@ -2,22 +2,66 @@
...
+Model Fitting
 =============
 
-More words...
+The Model Fitting plugin is described in more detail by the
+Specviz:Data Analysis Plugins:Model Fitting documentation.
+For Cubeviz, there is an additional option to fit the model over each individual spaxel by
+pressing :guilabel:`Apply to Cube`.  The fit parameter planes are saved in a data structure
+that can be accessed in a jupyter notebook. The best model fit, evaluated over the cube is
+also saved to a 'Model' dataset.
```
Comment (rosteen) [inline docs/cubeviz/plugins.rst:59]:
```suggestion
also saved to a dataset with the label specified in the :guilabel:`Model Label` field (default 'Model').
```

**Review by rosteen — CHANGES_REQUESTED** [review]
One minor suggestion, other than that it looks good. I think I'll plan to make a follow-up PR that enables the `autosectionlabel` sphinx plugin so that the references to the Specviz doc pages in the text can also be links.

**Review by javerbukh — APPROVED** [review]
Looks good, nice work! I think once glue-jupyter has the contours in we can quickly throw in some information about how to use that, but that can be after this is merged.

**Review by rosteen — APPROVED** [review]

**Comment by javerbukh** [comment]
LGTM!

**Pull request merged**